src/backend/distributed/worker/worker_data_fetch_protocol.c

```diff
@@ -787,7 +790,25 @@ worker_append_table_to_shard(PG_FUNCTION_ARGS)
 
 	sourceQualifiedName = quote_qualified_identifier(sourceSchemaName, sourceTableName);
 	sourceCopyCommand = makeStringInfo();
-	appendStringInfo(sourceCopyCommand, COPY_OUT_COMMAND, sourceQualifiedName);
+
+	/*
+	 * Partitioned tables do not support "COPY table TO STDOUT". Thus, we use
```

Comment:
Do we need this ? Even postgres does not support the usage and recommends `copy (select. ...) ...`

Reply:
I think I couldn't get your suggestion here.
If we don't do this, Postgres errors out when we send `COPY partitioned_table_shard ..` command, that's why I added this.

Reply:
right. my comment was total bs then.